driver-examples/src/main/java/com/datastax/driver/examples/basic/Example0001.java

```diff
@@ -0,0 +1,80 @@
+/*
+ *      Copyright (C) 2012-2015 DataStax Inc.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+package com.datastax.driver.examples.basic;
+
+import com.datastax.driver.core.Cluster;
+import com.datastax.driver.core.ResultSet;
+import com.datastax.driver.core.Row;
+import com.datastax.driver.core.Session;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * This example shows how to connect to a Cassandra cluster and
+ * extract some basic information from it.
+ *
+ * @see <a href="http://datastax.github.io/java-driver/manual/">Java driver online manual</a>
+ */
+public class Example0001 {
```

Comment:
It would be nice if the class names gave some indication of what they do:

* Example0001 => ReadCassandraVersion
* Example0002 => ReadHostsAndTables
* Example0003 => CreateAndPopulateSchema

Numbering the examples by increasing order of complexity is compelling, but it might not scale well: if the counter is at 500 and we realize a simple example is missing, there will be no way to insert it.